src/components/ControlInput.vue

```diff
@@ -93,7 +82,22 @@
                         />
                     </div>
                 </transition>
+                <div
+                    :class="{'kiwi-controlinput-tools-container-expand--inverse': !showPlugins}"
+                    class="kiwi-controlinput-tools-container-expand"
+                    @click="showPlugins=!showPlugins"
+                >
+                    <i v-if="!showPlugins" class="fa fa-ellipsis-h" aria-hidden="true"/>
+                    <i v-else class="fa fa-caret-right" aria-hidden="true" />
+                </div>
             </div>
+
+            <button v-if="shouldShowSendButton"
+                    class="kiwi-controlinput-send-button"
```

Comment:
You've added the new `kiwi-controlinput-send-button` class but the existing `kiwi-controlinput-send` classes still exist and even modified.

It would be better to keep `kiwi-controlinput-send` instead just incase anyone has plugins making use of `kiwi-controlinput-send` already. Don't forget to remove the obsolete CSS styles too

Reply:
I didn't see this change request, shall do first thing Monday morning!